src/components/tree/reactD3Interface/change.js

```diff
@@ -47,6 +55,20 @@ export const changePhyloTreeViaPropsComparison = (mainTree, phylotree, oldProps,
   /* explode! */
   if (oldProps.explodeAttr !== newProps.explodeAttr) {
     args.changeNodeOrder = true;
+    args.newFocus = newProps.focus;
+  }
+
+  /* enable/disable focus */
+  if (oldProps.focus !== newProps.focus) {
+    args.newFocus = newProps.focus;
+    args.updateLayout = true;
+  }
+  /* re-focus on changes */
+  else if (oldProps.focus == true &&
+           newProps.focus == true &&
+           (zoomChange || dateRangeChange || filterChange)) {
+    args.newFocus = true;
+    args.updateLayout = true;
```

Comment:
The naming of "newFocus" isn't quite right I think - sometimes it represents a change in focus state, but in the other 2 cases it simply represents the existing (unchanged) state as we need to pass this value to the `setDisplayOrder` function downstream. I'd suggest "focusState" or something.

(I know this is minor, but this section of code is relatively complex and when we re-read it months down the track little things like this become very helpful.)